apps/protailwind/src/components/exercise-overlay.tsx

```diff
@@ -106,15 +111,18 @@ const Actions = () => {
 }
 
 const ExerciseOverlay: React.FC<{tutorialFiles: any}> = ({tutorialFiles}) => {
-  const {lesson, module} = useMuxPlayer()
-  const {github} = module
-  const sandpack = lesson.sandpack
+  const {lesson} = useLesson()
+  const router = useRouter()
+  const {data: resources} = trpc.resources.byExerciseSlug.useQuery({
+    slug: router.query.exercise as string,
+    type: lesson._type,
+  })
```

Comment:
product specific resources are loaded via trpc. in this case it's `github`, `sandpack`, and `figma`